bsc-plugin/src/lib/rooibos/TestGroup.ts

```diff
@@ -58,10 +58,11 @@ export class TestGroup extends TestBlock {
         try {
             let func = this.testSuite.classStatement.methods.find((m) => m.name.text.toLowerCase() === testCase.funcName.toLowerCase());
             func.walk(brighterscript.createVisitor({
-                ExpressionStatement: (expressionStatement) => {
+                ExpressionStatement: (expressionStatement, parent, owner) => {
                     let callExpression = expressionStatement.expression as CallExpression;
                     if (brighterscript.isCallExpression(callExpression) && brighterscript.isDottedGetExpression(callExpression.callee)) {
                         let dge = callExpression.callee;
+                        let isSub = isFunctionExpression(callExpression.parent.parent.parent) && callExpression.parent.parent.parent.functionType.kind === TokenKind.Sub;
```

Comment:
This might night be good enough. What about assertions inside of statements? 

You probably need to walk upwards recursively until you hit a functionExpression, rather than assuming these are at always at the root. 

owningFunc = n.findAncestor(isFunctionExpression)

Reply:
ah.. gotcha.. yes indeed. will address, asap..